scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,127 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m venv .env
+#   source .env/bin/activate
+#   pip install --upgrade pip
+#   python3 -m pip install requests
+#   
+#   Authorization: Access the password in the vault using the appropriate command:
+#   vault login -method=github token=$(cat ~/.github-token)
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/user
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/userAdmin
```

Comment:
Could you add a note to make it clear that the password is needed for the auth dialog that the script will put up? Also this prints both passwords but I think only `userAdmin` is needed.

Also maybe add a note on how the env is used here. If you're running this for another environment, you'd have to change a few paths in this script.